Classes/IndexQueue/Initializer/AbstractInitializer.php

```diff
@@ -30,6 +30,7 @@
 use ApacheSolrForTypo3\Solr\Domain\Index\Queue\QueueItemRepository;
 use ApacheSolrForTypo3\Solr\Domain\Site\Site;
 use ApacheSolrForTypo3\Solr\System\Logging\SolrLogManager;
+use ApacheSolrForTypo3\Solr\Util;
```

Comment:
For master we could remove the usage.
We need it in backport against release-11.0.x branch.

Reply:
The use statement was removed.